### PR TITLE
[WIP] Fix transport issues

### PR DESCRIPTION
**Pull request by StanczakDominik**
Fixes a bunch of them. I'll update this post.

### CODE REVIEW

File: plasmapy/physics/transport/tests/test_collisions.py [hunk 1]
```diff
@@ -38,6 +38,12 @@ def setup_class(self):
         self.gms6 = 3.635342040477818
         self.gms6_negative = 0.030720859361047514
 
+    def test_symmetry(self):
+        lnLambda = Coulomb_logarithm(self.temperature1, self.density2, self.particles)
+        lnLambdaRev = Coulomb_logarithm(self.temperature1, self.density2, self.particles[::-1])
+        assert lnLambda == lnLambdaRev
```
Comment (lemmatum) [inline plasmapy/physics/transport/tests/test_collisions.py]: add error string for test failure

File: plasmapy/physics/transport/tests/test_collisions.py [hunk 2]
```diff
@@ -482,19 +475,24 @@ def setup_class(self):
         self.V = 1e4 * u.km / u.s
         self.True1 = 7.200146594293746e-10
 
+    def test_symmetry(self):
+        result = impact_parameter_perp(self.T, self.particles)
+        resultRev = impact_parameter_perp(self.T, self.particles[::-1])
+        assert result == resultRev
```
Comment (lemmatum) [inline plasmapy/physics/transport/tests/test_collisions.py]: error string

File: plasmapy/physics/transport/tests/test_collisions.py [hunk 3]
```diff
@@ -532,6 +530,11 @@ def setup_class(self):
         self.V = 1e4 * u.km / u.s
         self.True1 = np.array([7.200146594293746e-10, 2.3507660003984624e-08])
 
+    def test_symmetry(self):
+        result = impact_parameter(self.T, self.n_e, self.particles)
+        resultRev = impact_parameter(self.T, self.n_e,  self.particles[::-1])
+        assert result == resultRev
```
Comment (lemmatum) [inline plasmapy/physics/transport/tests/test_collisions.py]: error string

File: plasmapy/physics/transport/tests/test_collisions.py (hunk 1 above)
Comment (StanczakDominik) [reply plasmapy/physics/transport/tests/test_collisions.py]: I honestly think we can skip that! Here, let me show you what happens if I mess up this test by taking the other temperature times 20. This is part of my pytest output:
```python
========================================= FAILURES ==========================================
___________________________ Test_Coulomb_logarithm.test_symmetry ____________________________

self = <plasmapy.physics.transport.tests.test_collisions.Test_Coulomb_logarithm object at 0x7f7978f4d198>

    def test_symmetry(self):
        lnLambda = Coulomb_logarithm(self.temperature1, self.density2, self.particles)
        lnLambdaRev = Coulomb_logarithm(self.temperature1*20, self.density2, self.particles[::-1])
>       assert lnLambda == lnLambdaRev
E       assert 0.03193239302349768 == 3.8745137152918026

plasmapy/physics/transport/tests/test_collisions.py:44: AssertionError
```

That seems clean and understandable enough to me! Sure, the `self = object at hex address in memory` part is unhelpful, but the test is simple enough to see where you need to start digging without further prompting.

Comment (lemmatum) [reply plasmapy/physics/transport/tests/test_collisions.py]: Fair point, here it is not so ambiguous what is going on. It's just that in general our variable names aren't necessarily clear as to what is the expected value (and why) and what is the value obtained from the function.

Comment (StanczakDominik) [reply plasmapy/physics/transport/tests/test_collisions.py]: That just means we have to write better tests!

Comment (lemmatum) [reply plasmapy/physics/transport/tests/test_collisions.py]: You mean like clearly documenting what is going on in the form of an error string? :grin:

Comment (StanczakDominik) [reply plasmapy/physics/transport/tests/test_collisions.py]: I think well written and well named tests can actually be self.documenting. :smiley:

**Pull request merged**